tests/test_location_pi.py

```diff
@@ -427,10 +552,41 @@ def test15_get_stock_drift_no_package_token_with_lot(self):
 
         self.assertTrue(lot.exists(), "New lot does not exist")
         self.assertEqual(lot.name, lot_name, "New lot has an unexpected name")
-        self.assertEqual(lot.product_id.id, self.apple.id,
+        self.assertEqual(lot.product_id.id, self.strawberry.id,
                          "New lot is linked to an unexpected product")
 
-    def test16_process_inventory_adjustment_mss(self):
+    def test20_get_stock_drift_no_package_token_with_existent_lot(self):
+        """
+        Returns the correct metadata for the picking creation if
+        a request item contains a 'NO_PACKAGE' package and an
+        existent lot is specified.
+        """
+        Lot = self.env['stock.production.lot']
+
+        # NB: a quant for a serial tracked item should have the
+        # lot_id set and qty = 1
+        lot_name = "test_20_lot"
+        lot = Lot.get_lot(lot_name, self.strawberry.id, create=True)
+        self.create_quant(self.strawberry.id, self.test_location_01.id, 1,
+                          lot_id=lot.id)
+        adjs_req = [{"product_id": self.strawberry.id,
+                     "package_name": "test20_NO_PACKAGE_with_existent_lot",
+                     "quantity": 98,
```

Comment:
Why 98? if it is a serial tracked product this should be 1 or 0. Since we start with 1, we migth want to adjust it to 0.